Contents/Sketch/typography.js

```diff
@@ -7,7 +7,10 @@ var applyTypography = function (newStyles, sharedStyles) {
     'justified': 3
   };
 
-  removeAllStyles();
```

Comment:
I think I figured out. I'll be pushing a new branch for you to test out in a few minutes.